app.py

```diff
@@ -0,0 +1,591 @@
+import os
+import math
+import json 
+import datetime
+
+# external
+from flask import Flask, request, jsonify
+from flask_socketio import SocketIO, send, emit, join_room, leave_room
+from flask_sqlalchemy import SQLAlchemy
+from flask_marshmallow import Marshmallow
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    Boolean,
+    ForeignKey,
+    DateTime,
+    Sequence,
+    Float,
+)
+import jwt
+
+import warnings
+# https://github.com/marshmallow-code/flask-marshmallow/issues/53
+with warnings.catch_warnings():
+     from flask_marshmallow import Marshmallow
+
+def debug(msg):
+    print('\n\n\n'+msg+'\n\n\n')
+
+
+app = Flask(__name__)
+basedir = os.path.abspath(os.path.dirname(__file__))
+
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///' + \
+    os.path.join(basedir, 'db.sqlite')
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+app.config['SECRET_KEY'] = '\xfa\xfe\xc5\x8d \xaf\xbe\xb8`\xbfz\xbc\x96Z\x9b\xc1\x9b]\xefB\x1a\x1b\xe7\xf4'
```

Comment:
create an .env file for fetching secrets and keys. Also never push any secrets on github!! For reference you can add .env.sample defining the env variables required in the application.

Reply:
Ok will do that. This secret key will be changed later. But now will add in env